### DISCUSSION TITLE
Readme code block font has unconventional shape for ampersand

### DISCUSSION THREAD

**Discussion by dtolnay** [rust-lang/crates.io#4616] [Feature Requests]
### Current Behavior

![Screenshot from 2022-03-04 17-29-27](https://user-images.githubusercontent.com/1940490/156862049-3cef07bb-6327-4d98-8544-1609abab834a.png)

### Expected Behavior

I feel that a different choice of font, with an ampersand shape that more closely matches the majority of programming fonts, would be more approachable.

![georgia](https://user-images.githubusercontent.com/1940490/156862087-8d77df5f-4a60-4c4b-be70-596297d1989b.jpg)

### Steps To Reproduce

1. Go to any crate that has ampersand inside code block in the readme. Example: https://crates.io/crates/syn/1.0.86

### Environment

- Browser: Firefox 97.0
- OS: Ubuntu 20.04

**Comment by Turbo87** [comment]
we are currently using the ["Fira" font family](https://github.com/mozilla/Fira) for the whole page, including the code snippets in the Readmes

**Comment by dtolnay** [comment]
Fira Code has an official variant that uses sane ampersands. See ss03 in https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets. It might just need `font-feature-settings: "ss03"` in your CSS.

If that doesn't work, https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets#baking-in-stylistic-sets-into-the-font-file could be helpful to bake the ss03 ampersand into the font file.

If neither of those seems to be doing the trick, I think the ampersand is important enough to Rust that it's worth picking a different font over this.
